### DISCUSSION TITLE
"Speech_Check" during Map update after EU conversion

### DISCUSSION THREAD

**Discussion by AKakuba** [Mr-MIBonk/M.I.B._More-Incredible-Bash#415] [General]
as mentioned in the topic, I have converted mib2 from us to eu using m.i.b, the unit has the region and sds set to eu. fec has been added. And when installing eu maps, I get a "Speech_Check" message before the installation starts, which takes ~3700 seconds. After this time has elapsed, the message that the Speech_Check is not reachable appears. Do you have any idea what the reason for this might be and how to solve it?

**Comment by Busnic519** [comment]
> as mentioned in the topic, I have converted mib2 from us to eu using m.i.b, the unit has the region and sds set to eu. fec has been added. And when installing eu maps, I get a "Speech_Check" message before the installation starts, which takes ~3700 seconds. After this time has elapsed, the message that the Speech_Check is not reachable appears. Do you have any idea what the reason for this might be and how to solve it?

Did you manage to get over this? What did you do? I'm facing the same issue
